Dockerfile

```diff
@@ -9,6 +9,7 @@ WORKDIR /usr/src/app
 
 #Install Dependencies
 COPY package.json /usr/src/app
+RUN apt-get update && apt-get install -y git
```

Comment:
Let's move this instruction up. Otherwise, any time your package.json is changed you'll install git. If this line was high enough, the git installation phase would be taken from the docker cache.